outputs.tf

```diff
@@ -76,7 +76,7 @@ output "operator_private_ip" {
 
 output "ssh_to_operator" {
   description = "convenient command to ssh to the operator host"
-  value       = "ssh${local.ssh_key_arg} -J ${var.bastion_user}@${local.bastion_public_ip} ${var.operator_user}@${local.operator_private_ip}"
+  value       = "ssh${local.ssh_key_arg} -o ProxyCommand='ssh ${local.ssh_key_arg} -W %h:%p ${var.bastion_user}@${local.bastion_public_ip}' ${var.operator_user}@${local.operator_private_ip}"

```

Comment:
I think a space is needed after ssh `ssh${local.ssh_key_arg}`

Reply:
I left that how it was before I touched the code, it seems to output with a space so I assume `local.ssh_key_arg` has a space prefixed to it?

Either way it shouldn't effect the command if it has an extra space or not, let me know if you want me to add it and I can.

Reply:
I'm the original author of this file. I can provide the local ip